src/data_validation/data_schema.py

```diff
@@ -136,7 +137,7 @@ def create_toml(pdDict: dict) -> IO[str]:
     return toml_file
 
 
-test = read_DAP_csv("/ons/rdbe_dev/data_dictionary_berd.csv")
-test2 = convert_dataFrame(test)
-test3 = reformat_tomlDict(test2)
-test4 = create_toml(test3)
+csv_dataframe = read_DAP_csv("/ons/rdbe_dev/data_dictionary_berd.csv")
+csv_dataDict = convert_dataFrame(csv_dataframe)
+reformated_Dict = reformat_tomlDict(csv_dataDict)
+tomlFile = create_toml(reformated_Dict)
```

Comment:
Please check point 6 in this document on Python naming conventions: https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html

Reply:
These names are better, but still generic. "csv_dataframe" could refer to any dataframe that's come from a csv. Think about what the data contains. Also, to save space we often use `_df` as a suffix.

Reply:
Thank you for the link. I have gone over it and bookmarked it for future reference! 

I have also updated the naming on lines 140 - 143 to be more descriptive and match naming conventions. Hopefully they are better.